ls1intum/Artemis
# Bugfix/programming submission/submission state ui fixes

### Checklist

### Motivation and Context



### Description


Fix a couple of UI issues in the submission-state component:
- Corrected German translation for trigger-failed.button
- Improved tooltip text
- Only show result eta if at least 1 submission is running
- Lower debounce (UI updates more often)

### Steps for Testing


1. Log in to Artemis
2. Go to /#/exercise/`exerciseId`/participation of a programming exercise
3. Check the tooltips of the buttons (top right), the result eta should not be visible
4. Trigger the submission for a participation (e.g. by using the small trigger icon in the table)
5. The result eta should be visible until the submission receives a result

### Screenshots



![image](https://user-images.githubusercontent.com/28230611/64721060-0a231900-d4cc-11e9-8cb0-ecdf5f7163eb.png)
